File: 02_EstructurasControl/Secuenciales/Semana3_Secuenciales/Actividad Semanal/C#/Ejercicio_ActividadSemanal_5/Ejercicio_ActividadSemanal_5.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ejercicio_ActividadSemanal_5
{
    internal class Program
    {
        static void Main(string[] args)
        {
            //Hallar el área y el volumen de un paralelepípedo.
            Console.WriteLine("Ingrese el lado de largo: ");
            double largo = double.Parse(Console.ReadLine());
            Console.WriteLine("Ingrese el ancho:");
            double ancho = double.Parse(Console.ReadLine());
            Console.WriteLine("Ingrese el alto:");
            double alto = double.Parse(Console.ReadLine());

            double area = 2 * ((largo * ancho) + (largo * alto) + (ancho * alto));
            double volumen = (largo * ancho * alto);

            Console.WriteLine("El area es de : " + area);
            Console.WriteLine("El volumen es de : " + volumen);
            Console.ReadKey();
        }
    }
}

File: 02_EstructurasControl/Secuenciales/Semana3_Secuenciales/Actividad Semanal/Python/Ejercicio_ActividadSemanal_5.py
# Hallar el área y el volumen de un paralelepípedo.

largo = float(input("Ingrese el largo: "))
ancho = float(input("Ingrese el ancho: "))
alto = float(input("Ingrese el alto: "))

area = 2 * (largo * ancho + largo * alto + ancho * alto)
volumen = largo * ancho * alto

print("El Área:", area)
print("El Volumen:", volumen)
